test/common/buffer/buffer_fuzz_test.cc

```diff
@@ -301,42 +300,42 @@ uint32_t bufferAction(Context& ctxt, char insert_value, uint32_t max_alloc, Buff
     }
     int pipe_fds[2] = {0, 0};
     FUZZ_ASSERT(::pipe(pipe_fds) == 0);
+    Network::IoSocketHandleImpl io_handle(pipe_fds[0]);
     FUZZ_ASSERT(::fcntl(pipe_fds[0], F_SETFL, O_NONBLOCK) == 0);
     FUZZ_ASSERT(::fcntl(pipe_fds[1], F_SETFL, O_NONBLOCK) == 0);
     std::string data(max_length, insert_value);
     const int rc = ::write(pipe_fds[1], data.data(), max_length);
```

Comment:
nit: write() returns ssize_t.
https://linux.die.net/man/2/write

Reply:
changed to ssize_t.